### ISSUE TITLE
Question: `PipsPager.LosingFocus`

### ISSUE THREAD

**Issue by MartinZikmund** [microsoft/microsoft-ui-xaml#5960]
I am trying to port `PipsPager` to Uno Platform and I am confused about the `LosingFocus` method in there: 

https://github.com/microsoft/microsoft-ui-xaml/blob/f0f9256072d03d06d08344b0edcf990c70d30884/dev/PipsPager/PipsPager.cpp#L567

How is it attached to the `LosingFocus` event? I can't find any reference to it. Is it some kind of C++ magic 🙂 ?

**Comment by ranjeshj** [comment]
@MartinZikmund Is it actually getting called ? looks like dead code to me. @beervoley  ?

**Comment by beervoley** [comment]
@MartinZikmund @ranjeshj https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.losingfocus?view=winrt-20348

**Comment by MartinZikmund** [comment]
@beervoley I know that this event exists, but how is this method hooked up to that event? For example for the `Loaded` event there is usually something like this in the constructor:

```csharp
Loaded({ this, &NavigationViewItemBase::OnLoaded });
```

But in this case there is nothing similar for `LosingFocus` 🤔

**Comment by beervoley** [comment]
> @beervoley I know that this event exists, but how is this method hooked up to that event? For example for the `Loaded` event there is usually something like this in the constructor:
> 
> ```cs
> Loaded({ this, &NavigationViewItemBase::OnLoaded });
> ```
> 
> But in this case there is nothing similar for `LosingFocus` 🤔

@MartinZikmund @ranjeshj yes, you're correct. But (in my head) I remember the event hitting when I added it, I can't get my ahead around why did I push it back then if it's not getting called...
Anyway, it's **dead code** and can be safely deleted. 
@MartinZikmund if you have time and desire, could you please delete the **dead code**, remove `m_ifNextFocusElementInside` variable as well as adjust the logic of `OnLostFocus` with removal of the former variable?
